editor/store/reducer.js

```diff
@@ -143,18 +143,18 @@ export const editor = flow( [
 
 	// Track undo history, starting at editor initialization.
 	partialRight( withHistory, {
-		resetTypes: [ 'SETUP_NEW_POST', 'SETUP_EDITOR' ],
+		resetTypes: [ 'SETUP_EDITOR_STATE', 'SETUP_EDITOR' ],
```

Comment:
Haven't tested yet, but possible to remove `SETUP_EDITOR` here now?

Reply:
probably